valhalla/proposals/models.py

```diff
@@ -103,6 +114,7 @@ class TimeAllocation(models.Model):
     semester = models.ForeignKey(Semester)
     proposal = models.ForeignKey(Proposal)
     telescope_class = models.CharField(max_length=20, choices=TELESCOPE_CLASSES)
+    instrument_name = models.CharField(max_length=200, choices=INSTRUMENT_NAMES)
```

Comment:
can the choices be dynamic? I.e. leave choices=() but in forms/admin that uses it get it dynamically from configdb?

Reply:
I'm not sure how we could pull this off. Django needs a set list of values for validation. We could make it a freeform field, but we would risk serious bugs from typos.